spec/models/importers/organisation_spec.rb

```diff
@@ -1,28 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe Importers::Organisation do
+  let(:one_content_item_response) { build_seach_api_response [content_id: 'content-id-1'] }
+  let(:two_content_items_response) { build_seach_api_response [{content_id: 'content-id-1'}, {content_id: 'content-id-2'}] }
+
   it 'queries the search API with the organisation\'s slug' do
-    response = double(body: {
-      results: [
-        {
-          some_field: 'some_value',
-        },
-      ]}.to_json)
-    expected_url = 'https://www.gov.uk/api/search.json?filter_organisations=MY-SLUG&count=99&fields=content_id'
-    expect(HTTParty).to receive(:get).with(expected_url).and_return(response)
+    expected_url = 'https://www.gov.uk/api/search.json?filter_organisations=MY-SLUG&count=99&fields=content_id&start=0'
+    expect(HTTParty).to receive(:get).with(expected_url).and_return(one_content_item_response)
 
     Importers::Organisation.run('MY-SLUG', batch: 99)
   end
 
   context 'Organisation' do
     it 'imports an organisation with the provided slug' do
-      response = double(body: {
-        results: [
-          {
-            some_field: 'some_value',
-          },
-        ]}.to_json)
-      allow(HTTParty).to receive(:get).and_return(response)
+      expect(HTTParty).to receive(:get).and_return(one_content_item_response)
```

Comment:
We are not mocking the get request, just the library. So, I just wanted to be sure that we are actually using the mock. As commented in the description the library (HTTParty) along with the way we are testing it will go away in a few weeks, so feels good enough for the time being.